_posts/en/meetings/2015-10-01-meeting.md

```diff
@@ -0,0 +1,136 @@
+---
+title: IRC meeting summary for 2015-10-01
+permalink: /en/meetings/2015/10/01
+name: 2015-10-01-meeting
+type: meeting
+---
+## Logs
+
+The full IRC-logs can be found [here](http://bitcoinstats.com/irc/bitcoin-dev/logs/2015/10/01#l1443726030.0).
+
+## Main topics    
+
+- Mempool limiting  
+- BIP68 + CHECKSEQUENCEVERIFY  
+- CLTV soft fork deployment  
+- libconsensus merge time window  
+
+
+## Mempool limiting
+
+### background
+
+When a transaction is relayed across the network it is held by the nodes in memory, until it gets into a block. All these transactions that sit in memory are called the memorypool or mempool for short.
+Like we could see during the spam-attack if there's a big back-log of transactions that couldn't make it in the blockchain this mempool can get pretty big resulting in nodes crashing.
+
+To stop this from happening devs are trying to find a way to limit this mempool, so a mechanism to reject and/or remove transactions from the mempool. The hard part here is to make it so nodes can't be attacked by abusing this mechanism.
+
+There are multiple worked out ideas for this, namely:   
+[Limit mempool by throwing away the cheapest txn and setting min realy fee to it](https://github.com/bitcoin/bitcoin/pull/6722)  
+[Mempool limiting with descendant package tracking](https://github.com/bitcoin/bitcoin/pull/6557)  
+[exponential rising effective min relay feerate](https://github.com/bitcoin/bitcoin/pull/6673)
```

Comment:
I get the feeling the above should be a list, or should at least be a comma-separated serialized list.